packages/components/molecules/f-rating/package.json

```diff
@@ -45,7 +45,8 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.x"
+    "@justeat/f-services": "1.x",
+    "@justeattakeaway/pie-icons-vue": "1.x"
```

Comment:
Did you try using the beta package for the icons?

Reply:
Should we be using the beta? Happy to swap over.

Reply:
Yes please – it's the same package, it's just in beta while we wait for the name changes from design.

If it works ok in this though, I'll change the readme of the old package to point to the new one

Reply:
Added.